MiniProject 1/VideoViewController.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  VideoViewController.swift
+//  MiniProject 1
+//
+//  Created by ESIEA on 10/04/2015.
+//  Copyright (c) 2015 HaiNguyen. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+
+class VideoViewController: UIViewController
+{
+    var moviePlayer: AVPlayer?
+    var playerLayer: AVPlayerLayer?
+    var playerItem: AVPlayerItem?
+    let screenSize = UIScreen.mainScreen().bounds
+    var urls = ["http://fun.siz.io/stories/142893791787803c7fb48f4d/0.mp4",
+                "http://fun.siz.io/stories/142893791787803c7fb48f4d/1.mp4",
+                "http://fun.siz.io/stories/142893791787803c7fb48f4d/2.mp4",
+                "http://fun.siz.io/stories/142893791787803c7fb48f4d/3.mp4"]
+    
+    var videoNumber = 0
+    
+    func displayButton()
+    {
+        let leftButton = UIButton.buttonWithType(UIButtonType.System) as! UIButton
+        leftButton.frame = CGRectMake(0, (screenSize.width/2)-20, 80, 40)
+        leftButton.backgroundColor = .blackColor()
+        leftButton.setTitle("Prev", forState: UIControlState.Normal)
+        leftButton.setTitleColor(UIColor.whiteColor(), forState: UIControlState.Normal)
+        leftButton.addTarget(self, action: "leftButtonAction:", forControlEvents: UIControlEvents.TouchUpInside)
+        
+        let rightButton = UIButton.buttonWithType(UIButtonType.System) as! UIButton
+        rightButton.frame = CGRectMake((screenSize.height)-80, (screenSize.width/2)-20, 80, 40)
+        rightButton.backgroundColor = .blackColor()
+        rightButton.setTitle("Next", forState: UIControlState.Normal)
+        rightButton.setTitleColor(UIColor.whiteColor(), forState: UIControlState.Normal)
+        rightButton.addTarget(self, action: "rightButtonAction:", forControlEvents: UIControlEvents.TouchUpInside)
+        
+        self.view.addSubview(leftButton)
+        self.view.addSubview(rightButton)
+    }
+    
+    func leftButtonAction(sender:UIButton!)
+    {
+        moviePlayer!.pause()
```

Comment:
Do not force optionnal, use "let if" or myObject?myMethods() instead for a crash proof code.